src/test/java/com/rultor/agents/github/qtn/QnReleaseTest.java

```diff
@@ -77,4 +79,48 @@ public void buildsRequest() throws Exception {
         );
     }
 
+    /**
+     * QnRelease can deny release when tag is outdated.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void denyOutdatedTag() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        repo.releases().create("1.7");
+        issue.comments().post("release `1.6`");
+        MatcherAssert.assertThat(
+            new QnRelease().understand(
+                new Comment.Smart(issue.comments().get(1)), new URI("#")
+            ),
+            Matchers.is(Req.EMPTY)
+        );
+        MatcherAssert.assertThat(
+            new Comment.Smart(issue.comments().get(2)).body(),
+            Matchers.containsString("There is already a release `1.7`")
+        );
+        issue.comments().post("release");
+    }
+
+    /**
+     * QnRelease can deny release when tag name is not given.
```

Comment:
@mkordas you know the weird way "release questions" are answered at present Rultor does not answer when the backticks with the version inside are missing. I think somewhere down the line an Exception is thrown and then swallowed in this case. There is a ticket about this somewhere and you can also observe the effect of this when you as a non-architect call the release command. It always posts a second "I don't understand you answer", so far me it seemed safest to cover this case in a test and in code here and go from there with the other ticket.
Also it simply seemed the cleanest approach to just handle this case instead of removing the old test, or leaving in this as a strange corner case that no one understands.
To me it makes sense, I'm open to criticism though ? :)